GAME/LARAFIRE.C

```diff
@@ -3,6 +3,7 @@
 #include "GAMEFLOW.H"
 #include "SPECIFIC.H"
 #include "OBJECTS.H"
+#include "LARA.H"
```

Comment:
Includes must be ordered alphabetically.

Reply:
fixed